app/Filament/Resources/GameResource.php

```diff
@@ -445,6 +448,11 @@ public static function table(Table $table): Table
                         ->icon('fas-clock-rotate-left'),
                 ]),
             ])
+            ->recordUrl(
+                fn (Game $record) => $user->can('update', $record)
+                    ? GameResource::getUrl('edit', ['record' => $record])
+                    : GameResource::getUrl('view', ['record' => $record])
+            )
```

Comment:
I'm not sure there's a correct solution for this. Some users might be going to the game specifically to edit it, others might be looking for summary information.

There's information on the non-edit page (like metrics) which may be what the user is looking for, so redirecting them to the edit page becomes extra work. Also, it's less clear that clicking "details" will leave the edit page. At least, it's clear that clicking "edit" will go to the edit page.

Similarly, Should the achievements/leaderboards/hashes/code notes be present on the edit game page since you're able to edit them without opening the edit page?